Core/StitchCoreSDK/Sources/StitchCoreSDK/Auth/Internal/CoreStitchUser.swift

```diff
@@ -32,6 +39,11 @@ public protocol CoreStitchUser {
      * to this user which can be used to log in as this user.
      */
     var identities: [StitchUserIdentity] { get }
+
+    /**
+     * The last time that this user was logged into, switched to, or switched from
+     */
+    var lastAuthActivity: Double { get }
```

Comment:
[nit] I think it would be a better UX for users to make this a `Date`, and internally store the time since Unix Epoch

Reply:
Either way, it shouldn't be a Double. It should either be a `Date` or a `TimeInterval`